### ISSUE TITLE
Incorrect y-axis time values in helicorder examples

### ISSUE THREAD

**Issue by rbeerster** [crotwell/seisplotjs#32]
In both helicorder examples, [https://crotwell.github.io/seisplotjs/gallery/helicorder.html](https://crotwell.github.io/seisplotjs/gallery/helicorder.html) and [https://crotwell.github.io/seisplotjs/examples/helicorder/index.html](https://crotwell.github.io/seisplotjs/examples/helicorder/index.html), the y-axis time values (yLabel and yLabelRight) are incorrect. Here is a diff from the .mjs file for  seismographconfig.ts file that made it work, but I am not confident that this is the correct fix:
```
53951c53951
<     return this.__cache__.yLabelHandlebarsCompiled(context, runtimeOptions);
---
>     return  this.__cache__.yLabelHandlebarsCompiled = Handlebars.compile(this._yLabel);
53993,53995c53993
<         this.__cache__.yLabelRightHandlebarsCompiled = Handlebars.compile(
<           this._yLabelRight
<         );
---
>         this.__cache__.yLabelRightHandlebarsCompiled = Handlebars.compile(this._yLabelRight);
54001c53999,54000
<     return this.__cache__.yLabelRightHandlebarsCompiled(context, runtimeOptions);
---
>     // return this.__cache__.yLabelRightHandlebarsCompiled(context, runtimeOptions);
>     return this.__cache__.yLabelRightHandlebarsCompiled = Handlebars.compile(this._yLabelRight);
```
I hope this helps track down the real issue.

**Comment by crotwell** [comment]
a cache was not being cleared, believe fixed in 2b14fb11163ea20e5af790cf2f35d6601034306e but please test 

thanks for bug report

**Comment by rbeerster** [comment]
This seems to have corrected the yLabel and yLabelRight values.
